bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/LLJJ_105-160_EWK_SM_5f_LO/LLJJ_105-160_EWK_SM_5f_LO_customizecards.dat

```diff
@@ -0,0 +1,33 @@
+# These inputs are based on fixing the PDG vlaues of mZ, mW, and G_fermi
+# The MC programs use a constant width scheme, whereas PDG values are sqrt(s) dependent.
+# We correct for this following Sect. 3.1 in https://arxiv.org/pdf/1606.02330.pdf
+
+set param_card SMINPUTS 1 128.82531590804655
+set param_card SMINPUTS 2 1.1663787e-05
+
+# Z boson
+set param_card mass 23 91.153509740726733
+set param_card DECAY 23 2.4932018986110700
+
+# W boson
+set param_card mass 24 79.906853549493746
+set param_card DECAY 24 2.085
+
+# https://twiki.cern.ch/twiki/bin/view/LHCPhysics/LHCTopWGSummaryPlots#Top_Quark_Mass
+# CMS combined result (Sep 2015)
+
+# top quark
+set param_card mass 6 172.5
+set param_card DECAY 6 1.311
+set param_card yukawa 6 172.5
+
+# User settings
+set run_card mmjj 120.0
+set run_card mmll 105.0
+set run_card mmllmax 160.0
+set run_card auto_ptj_mjj False
+set run_card ptj 5.0
+set run_card ptb 5.0
```

Comment:
have you checked that after mmjj cut, ptj and ptb is already much higher than 5GeV used here

Reply:
Hi @menglu21, Here is the comparison with and without the pTj and pTb 5GeV cut. If we remove the ptJ and ptb 5GeV cut, about 3,100/10,000 events would be into the first bin [0,5]GeV. And the XS is also different 0.0635 +- 0.0003071 pb VS 0.08862 +- 0.0005419 pb. more details please check the attached plots and gridpack log files.

https://qguo.web.cern.ch/qguo/Hmumu/LLJJ_nojpT5/
https://qguo.web.cern.ch/qguo/Hmumu/LLJJ_jpT5/

[pT_j_nojpT.png](https://qguo.web.cern.ch/qguo/Hmumu/LLJJ_nojpT5/pT_j_nojpT.png)
[pT_j_jpT5.png](https://qguo.web.cern.ch/qguo/Hmumu/LLJJ_jpT5/pT_j_jpT5.png)

Reply:
thanks, then won't use high ptj&ptb cut be better, cuz in analysis level you will require 20 or 30 GeV on jet. Use high cut can let you have more effective events produced

Reply:
@rgerosa thanks for the elaboration, since you already studied the effect, it's fine for me to merge it.